src/accelerate/big_modeling.py

```diff
@@ -302,6 +302,7 @@ def dispatch_model(
     offload_dir: Optional[Union[str, os.PathLike]] = None,
     offload_index: Optional[Dict[str, str]] = None,
     offload_buffers: bool = False,
+    cpu_offload: bool = False,
```

Comment:
In general, should newly added parameters be placed last in case someone calls this function with purely positional arguments?